### ISSUE TITLE
sandy mission 2-1 freezeing on npc interaction

### ISSUE THREAD

**Issue by thenoobers** [LegionDark/Issues#1641]
**Date & Time:**: 
9/28/2017 @ approx 1500 PST

**FFXI Client Version (use `/ver`)**: 
30170905_1

**Server's Expected Client Version matches, yes/no? (use `$ecv`)**: 
R:  3017070

**Character Name**: 
Noobers

**Nation**: 
Sandoria

**Job(level)/Sub Job(level)**: 
99blm/49 rdm

**NPC or Monster or item Name**: 

Tayula (prompts CS feeds lines for did you find anything?  second line - Hey!  and freezes right there 3rd possible npc to obtain bronze sword needed to trade to npc in ordelles caves.

**Zone name**: 
La Thein Platue

**Coordinates (use `$where`)**: 
la thein platue
X:-300   Y:31    Z:147 

**ffxiah.com link (for items issues only)**: 

**Multi-boxing? (multiple clients on same connection)**: 
Yes i was multi boxing at the time

**Steps To Reproduce / any other info**: 
sandy mission 2-1 rescue mission most specifically the NPC 
i spoke to narvecaint ontop of the hill by the wall after being prompted by immortal that i missed him.  than walked down and spoke to npc in ordelles caves behind the rocks.  went back out to la thein and talked to the npcs on the road and narvecaint again with no sword being obtained.  than talked to all the npcs along the path 2 prompt me with did you find anything.  nope nothing.  or keep looking.  and when i talk to Tayula shes the 3rd npc which gives me the 3rd CS like dialogue but when the 2nd dialogue line hits Hey!  it just freezes.  i can still type in game but its CutScene locked i cant open any menus move cancel or do anything.

**Comment by EpicTaru** [comment]
So per our conversation in-game as well as what you provided here, the most likely culprit for your issue is the version mismatchs between your client and the server.  Your client is currently running 30170905 whereas the server is running 30170707.  It's not your fault that the version mismatch occurred, it just happens usually when retail updates their version, which POL updates clients to when you perform the File Check in POL, nothing anyone can do about that, unfortunately.  The server version won't be updated until DSP releases an updated version of the DSP core files.  At that time, PS (private server) owners will grab the core files and perform a version update of their PS.

TLDR: Version mismatches can cause ID shifts in the names of NPCs, which prevents some missions/quests/CSes from working. You just gotta be patient and wait for LegionDark to update the server version, which has to wait until DSP performs their part of the process.

**Comment by thenoobers** [comment]
there's no way someone can force cancel my mission so i can change allegiance?  this is my first missions run and its topping me from getting into dynamis and progressing.

**Comment by Heepster03** [comment]
Being that this is a client mismatch issue, there is a good chance that you'll get the issues no matter which nation that you choose to go with. That's why we always tell people that you shouldn't update due to the fact that it'll cause issues like this. And for those that can't help it and have to be on a newer version, just gotta wait it out.

**Comment by thenoobers** [comment]
I actually completed bastok upto 5-1 on a mithra and by update to the new version you mean follow the installation instructions per the website wiki? Thats the only update i did.  Im hopeful an update will come and lift restraints from my elvaan blm one day.  But untill than just keep up the good work guus i have faith.  And time to play other chars haha!

**Comment by thenoobers** [comment]
i do have a serious question that when you said version and updated to the new version.  kinda got me to think.  When I DL from SquareEnix.  I should only DL  FFXI CORE   and POL  as pre your site right.  even though its offering the whole expansions yadi.  so if I by accident had select to install all versions.  would this be updating to the new version like what your saying.   IF   that is what happened.   CAN I????  simply just uninstall ffxi and spend 8 hours doing it over again CORRECTLY?  so that I can continue or will I now be forever glitched because I prompted in the wrong version?  just an idea gimme some feedback!!!!  and thankyou for helping me thus far!

**Comment by TeoTwawki** [comment]
cant update from installer ver (which is years out if date) to anything except the latest retail. expantikns have nothing to do with it either. nothing to do but wait for the server ti catch up
